smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -382,7 +382,7 @@ private void bindCommandsToCoDriverXboxButtons()
         this.coDriverController.rightTrigger().onTrue(
```

Comment:
you need an import at the top of this file

Reply:
done